html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title> HTML Forms </title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+
+    <p>Email</p>
+    <label for="email">
+      <input type="text" id="email" name="email">
+    </label>
+
+    <p>Why is coding cool?</p>
+    <label for="codingq">
+      <select id="codingq" name="codingq">
+        <option selected=""> Select an option</option>
+        <option value="interesting"> Because it's interesting</option>
+        <option value="fun"> Because it's fun</option>
+        <option value="powerful"> Because it's powerful</option>
+      </select>
+    </label>
+
+    <p>Favorite Language (choose one)</p>
+    <input id="HTML" type="radio" name="language" value="HTML">
+    <label for="HTML"> HTML </label>
+    <input id="CSS" type="radio" name="language" value="CSS">
+    <label for="CSS"> CSS </label>
+    <input id="JavaScript" type="radio" name="language" value="JavaScript">
+    <label for="JavaScript"> JavaScript </label>
+
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="signUp">
+    <label for="signUp"> Sign me up! </label>
+
+    <p>Comments</p>
```

Comment:
This should be the `label` element, it is the text labelling the form control element.